rest/src/main/java/com/carhouse/rest/controller/CarSaleController.java

```diff
@@ -48,6 +50,18 @@ public CarSaleController(final CarSaleService carSaleService) {
         return carSaleService.getCarSales();
     }
 
+    /**
+     * Send all car sales without redundant information to show list of them.
+     *
+     * @param requestParams the request params
+     * @return the list of car sales in JSON
+     */
+    @GetMapping("/dto")
```

Comment:
Why this endpoint is so important and required? Can't understand difference with `/carSales`.